src/engraving/rendering/dev/pagelayout.cpp

```diff
@@ -225,6 +226,10 @@ void PageLayout::collectPage(LayoutContext& ctx)
                 collected = true;
             }
         }
+
+        float curPercent = static_cast<float>(ctx.state().measureNo()) / ctx.dom().lastSegment()->measure()->index();
```

Comment:
Worth checking that lastSegment() is not null

Reply:
fixed